samples/testing/operations/test_operations.py

```diff
@@ -66,7 +57,6 @@ def test_custom_operation() -> None:
     ]
 
 def test_operation_no_args_in_qsharp_file() -> None:
-    qsharp.init(target_profile=qsharp.TargetProfile.Unrestricted)
     qsharp.init(project_root='.')
```

Comment:
Same comment, since we have a fixture that does exactly the same thing, we don't need to repeat .init in the individual tests. Please remove duplicate inits